### ISSUE TITLE
Error when trying to build python wrapper 

### ISSUE THREAD

**Issue by Firsthero511** [xbpeng/DeepMimic#144]
when i try to execute `make python` i get this message:
`clang++ -c -g -std=c++11 -O3 -Wall -fPIC  -I./ -I../../libraries/eigen -I../../libraries/bullet3/src -I/usr/include/python3.6m -I/usr/lib/ -lpython3.6m  -o objs/Main.o Main.cpp
clang: warning: -lpython3.6m: 'linker' input unused [-Wunused-command-line-argument]
In file included from Main.cpp:3:
In file included from ./DeepMimicCore.h:3:
In file included from ./util/ArgParser.h:6:
./util/MathUtil.h:5:10: fatal error: 'Eigen/Dense' file not found
#include "Eigen/Dense"
         ^~~~~~~~~~~~~
1 error generated.
make: *** [Makefile:51: objs/Main.o] Error 1`

**Comment by xbpeng** [comment]
looks like it can't find the Eigen files. Make sure the include paths are correct in the Makefile.

**Comment by Firsthero511** [comment]
> looks like it can't find the Eigen files. Make sure the include paths are correct in the Makefile.

yes i was able to fix that, but now i am getting a python.h error, tried doing the same thing but it'd rewrite the file

**Comment by Danfoa** [comment]
@Firsthero511 same here. I am using anaconda and I am properly giving the python include folder path in the Makefile, however, it seems unable to find the Python.h file (which i checked is there obviously).

I am using anaconda and python 3.8

**Comment by Nish228** [comment]
> 

Hi, I had the same error and I was wondering if you found a solution?

**Comment by WilliamTambellini** [comment]
pull the latest master and try to run the (new) build.sh script.

**Comment by Touutae-lab** [comment]
what is the path look like  ? @Danfoa 
i'm using miniconda 
```
$ which python
/home/touutae/miniconda3/envs/tf/bin/python
```
